etc/orthography.tsv

```diff
@@ -0,0 +1,89 @@
+Grapheme	IPA	UNICODE	FREQUENCY
+^	NULL	NULL	NULL
+$	NULL	NULL	NULL
+(	NULL	NULL	NULL
+)	NULL	NULL	NULL
+*	NULL	NULL	NULL
+?	NULL	NULL	NULL
+~	NULL	NULL	NULL
+a	a	U+0061	503
+á	a	U+00e1	4
+ã	ã	U+00e3	282
+ɸ	ɸ	U+0278	13
+β	β	U+03b2	5
+c	c	U+0063	20
+ts	ts	U+0074 U+0073	9
+e	e	U+0065	371
+ẽ	ẽ	U+1ebd	133
+h	h	U+0068	216
+ʔ	ʔ	U+0294	231
+ʔh	ˀh	U+0294 U+0068	3
+ʔw	ʔʷ	U+0294 U+0077	2
+i	i	U+0069	402
+í	í/i	U+00ed	1
+ĩ	ĩ	U+0129	158
+ɨ	ɨ	U+0268	300
+ɨ́	ɨ	U+0268 U+0301	1
+ɨ̃	ɨ̃	U+0268 U+0303	129
+j	j	U+006a	224
+y	j		
+jʔ	jˀ		
+j̃	j̃	U+006a U+0303	2
+ʔj	ˀj		
+g	g	U+0067	32
+k	k	U+006b	330
+kh	kʰ	U+006b U+0068	23
+ki	kʲ		
+kj	kʲ		2
+kk	kː	U+006b U+006b	1
+kw	kʷ	U+006b U+0077	3
+kʔ	kˀ		
+ʔk	ˀk		
+kwʔ	kʷˀ		
+ʔkh	ˀkʰ		
+l	l	U+006c	4
+m	m	U+006d	296
+̃b	m		
+n	n	U+006e	157
+ŋ	ŋ	U+014b	18
+ɲ	ɲ	U+0272	63
+ʔn	ˀn		
+̃d	n		
+b	b	U+0062	113
+p	p	U+0070	194
+bʔ	bˀ		
+ph	pʰ	U+0070 U+0068	5
+pʔ	pˀ		
+ʔb	ˀb		
+r	ɾ	U+0072	61
+r̃	ɾ̃	U+0072 U+0303	17
+ʔr̃	ˀɾ̃		
+s	s	U+0073	207
+∫	ʃ	U+222b	24
+sʔ	sˀ		
+ʔs	ˀs	U+0294 U+0073	14
+d	d	U+0064	92
+t	t	U+0074	285
+th	tʰ	U+0074 U+0068	16
+tt	tː	U+0074 U+0074	1
+tʃ	tʃ	U+0074 U+0283	1
+tʔ	tˀ		
+t∫	tʃ		
+ʔd	ˀd		
+ʔt	ˀt		
+o	o	U+006f	397
+õ	õ	U+00f5	130
+v	w	U+0077	
+w	w	U+0077	106
+w̃	w̃	U+0077 U+0303	14
+~w	w̃	U+0077 U+0303	
+̃w	w̃	U+0077 U+0303	
+u	u	U+0075	182
+ú	ú/u	U+00fa	2
+ũ	ũ	U+0169	91
++	+	U+002b	11
+-	+		
+_	_	U+005f	4156

```

Comment:
We now render the `_` as `+` always, as we say that we cannot meaningfully segment words affixes, etc., so we just say: one segmenter of meaning units.

Reply:
ok